app/models/property.rb

```diff
@@ -65,6 +65,19 @@ def address_complete?
   end
 
   def property_owners_complete?
-    property_owners.all?(&:complete?)
+    return true if has_other_owners == YesNoAnswer::NO.to_s || has_other_owners.nil?
+
+    (has_other_owners == YesNoAnswer::YES.to_s) &&
+      property_owners.present? &&
+      property_owners.all?(&:complete?)
+  end
+
+  def property_ownership_valid?
```

Comment:
I think this logic is used in the property owners validator, would be good to see if there is a way to avoid duplicating the logic but don't worry if it's not straightforward

Reply:
I tried but its not a straightforward thing. 😢

Reply:
I thought so  😢